README.md

```diff
@@ -57,8 +57,55 @@ jobs:
         parameters: storageAccountType=Standard_LRS
 ```
 
-## Another example on how to use this Action to use get the output of ARM template
-In this exmaple, our template outputs `containerName`.
+## Another example which ensures the Azure Resource Group exists before ARM deployment
+In the preceeding example there is a pre-requisite that an existing Azure Resource Group named ```github-action-arm-rg``` must already exist.  
+
+The below example makes use of the [Azure Powershell Action](https://github.com/marketplace/actions/azure-powershell-action) to ensure the resource group is created before doing an ARM deployment.
+
+## Steps
+When generating your credentials (in this example we store in a secret named ```AZURE_CREDENTIALS```) you will need to specify a scope at the subscription level.
+
+```azurecli
+az ad sp create-for-rbac --name "{sp-name}" --sdk-auth --role contributor --scopes /subscriptions/{subscription-id}
+```
+
+See [Configure deployment credentials](https://github.com/marketplace/actions/azure-login#configure-deployment-credentials).
+
+## Example
+```yaml
+on: [push]
+name: AzureLoginSample
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    env:
+      ResourceGroupName: github-action-arm-rg
+    steps:
+    - uses: actions/checkout@master
+    - uses: azure/login@v1
+      with:
+        creds: ${{ secrets.AZURE_CREDENTIALS }}
+        enable-AzPSSession: true
+    - uses: Azure/powershell@v1
```

Comment:
Lets  add CLI action instead of Powershell.  Documentation suggests using CLI to create SPN

Reply:
@pulkitaggarwl 

It looks like at time of writing, this action is only available when using linux? I can update it, but this example wouldn't work for users opting to use for e.g. windows-latest though

![image](https://user-images.githubusercontent.com/13502806/108586720-1b58a080-73b5-11eb-94cd-d9dc47d22a0b.png)

thoughts?

Reply:
@pulkitaggarwl 

I updated the example so it uses Azure CLI though. Let me know if any other changes are required. Otherwise, if happy please resolve both active comments and merge when ready :)

Reply:
@kanika1894  can you look here.  how can we use Azure/CLI on Windows. If we can't we should update AzureCLI documentation

Reply:
@pulkitaggarwl cli action only supports Linux based runners. @kanika1894 can you please update the doc for cli action.